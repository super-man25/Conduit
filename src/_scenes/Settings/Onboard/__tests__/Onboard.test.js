import React from 'react';
import { shallow } from 'enzyme';
import { Onboard } from '../index';

describe('<Onboard />', () => {
  const props = {
    onboardActions: {}
  };

  it('renders correctly', () => {
    const wrapper = shallow(<Onboard {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
