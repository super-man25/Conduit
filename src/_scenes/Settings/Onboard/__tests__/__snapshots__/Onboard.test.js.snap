// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Onboard /> renders correctly 1`] = `ShallowWrapper {}`;
